hy/cmdline.py

```diff
@@ -1,10 +1,13 @@
+# fmt: off
```

Comment:
delete this

Reply:
I thought you were over this, or at least had decided that deviation from Black was acceptable in some circumstances, since you didn't object to some other PRs with `fmt: off`. What's going on?